adapters/package.json

```
@@ -12,6 +12,9 @@
     "build": "tsc -b ./tsconfig.build.json"
   },
   "dependencies": {
+    "ethers": "^5.4.6"
+  },
+  "devDependencies": {
     "@defichain/salmon-fetch": "0.0.0"
```

```suggestion
    "@defichain/salmon-fetch": "0.0.0",
    "ethers": "^5.4.6"
```

`salmon-fetch` should be dependencies?

---

hmm.. i think move the `salmon-fetch` to peer dependencies??  :thinking:

---

They both need to be in `"dependencies"`, required for compilation

---

yea.. saw that error during build... moving back to dep

---

`ethers` need to be in dependencies too

---

ahh.. yea.. fixed!!